aws-secret-operator/templates/role.yaml

```diff
@@ -0,0 +1,34 @@
+{{- if .Values.scope.singleNamespace -}}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ include "aws-secret-operator.fullname" . }}
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  - services
+  - endpoints
+  - persistentvolumeclaims
+  - events
+  - configmaps
+  - secrets
```

Comment:
I think this is due to my fault in the upstream but this should ideally contain `secrets` only as I believe that's the only resource kind that the operator needs to access in the v1/core API group.

But I defer the decision of whether cleaning it up or keep it consistent with the upstream, to you 😃

Reply:
Sorry, the content was different between `cluserrole` and `role`.